### PR TITLE
startx: add page

### PR DESCRIPTION
**Pull request by nath1as**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/linux/startx.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# startx
+
+> The startx script is a front end to xinit that provides a somewhat nicer user interface for running a single session of the X Window System.
```
Comment (kbdharun) [inline pages/linux/startx.md:3]:
```suggestion
> The startx script is a front end to xinit that provides a nice user interface for running a single session of the X Window System.
```
Can we rephrase this a bit?
> Even though the man page has this description.

File: pages/linux/startx.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Start an X session with a predefined depth value:
+
+`startx -- -depth {{number}}`
+
+- Start an X session with a predefined dpi value:
+
+`startx -- -dpi {{number}}`
+
+- Override your .xinitrc settings and start a new X session:
```
Comment (kbdharun) [inline pages/linux/startx.md:18]:
```suggestion
- Override your `.xinitrc` settings and start a new X session:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution, I have some suggestions for this page.

File: pages/linux/startx.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`startx -- -depth {{number}}`
+
+- Start an X session with a predefined dpi value:
+
+`startx -- -dpi {{number}}`
+
+- Override your `.xinitrc` settings and start a new X session:
+
+`startx /path/to/{{window_manager | desktop_environment }}}`
```
Comment (kbdharun) [inline pages/linux/startx.md:20]: How about this?
```suggestion
`startx /{{path/to/window_manager_or_desktop_environment}}`
```
Similar to [path/to/file_or_directory](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md)

Or
```suggestion
`startx /{{path/to/window_manager | desktop_environment}}`
```

@mfrw @CleanMachine1 @navarroaxel What do you think about this?

File: pages/linux/startx.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+> The startx script is a front end to xinit that provides a nice user interface for running a single session of the X Window System.
+> More information: <https://x.org/releases/X11R7.5/doc/man/man1/startx.1.html>.
+
+- Start an X session:
+
+`startx`
+
+- Start an X session with a predefined depth value:
+
+`startx -- -depth {{number}}`
```
Comment (kbdharun) [inline pages/linux/startx.md:12]:
```suggestion
`startx -- -depth {{value}}`
```
I think the value would be better in this command argument.

File: pages/linux/startx.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+- Start an X session with a predefined dpi value:
+
+`startx -- -dpi {{number}}`
```
Comment (kbdharun) [inline pages/linux/startx.md:16]:
```suggestion
`startx -- -dpi {{value}}`
```

File: pages/linux/startx.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+- Start an X session with a predefined depth value:
+
+`startx -- -depth {{value}}`
+
+- Start an X session with a predefined dpi value:
+
+`startx -- -dpi {{value}}`
+
+- Override your `.xinitrc` settings and start a new X session:
```
Comment (navarroaxel) [inline pages/linux/startx.md:18]: We try to avoid `you`/`your`
```suggestion
- Override the settings in the `.xinitrc` file and start a new X session:
```

**Review by blueskyson — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Comment by kbdharun** [comment]
@CleanMachine1 was this PR title edited accidentally?
This is a new page and not a command addition, I will revert it back.

**Comment by CleanMachine1** [comment]
> @CleanMachine1 was this PR title edited accidentally? This is a new page and not a command addition, I will revert it back.

Yes it was, I loaded up the page and and saw it was just a commit I was viewing.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**